packages/ag-charts-website/src/components/homepage/components/hero/ImageCarousel.tsx

```diff
@@ -9,15 +8,6 @@ const ImageCarousel = () => {
     const rightColumnRef = useRef(null);
     const [scrollHeight, setScrollHeight] = useState(0);
 
-    const [darkMode] = useDarkmode();
```

Comment:
Not sure if we want the carousel images to be dark or not. If we want them to be dark, we can revert the last commit of this PR